picard/acousticbrainz/__init__.py

```diff
@@ -71,14 +74,15 @@ def ab_setup_extractor(tagger, config=None):
         version = check_extractor_version(acousticbrainz_extractor, tagger)
         if version:
             config.setting["use_acousticbrainz"] = True
-            config.setting["acousticbrainz_extractor_version"] = version
-            config.setting["acousticbrainz_extractor_sha"] = sha
+            ACOUSTICBRAINZ_EXTRACTOR_VERSION = version
+            ACOUSTICBRAINZ_EXTRACTOR_SHA = sha
             tagger.log.debug("AcousticBrainz is available")
-            return
+            return version

```

Comment:
Since version is returned, it seems to me `ACOUSTICBRAINZ_EXTRACTOR_VERSION` isn't needed.

Reply:
Yup, removing it.